src/OmniSharp/Api/ProjectSystem/OmnisharpController.CurrentProject.cs

```diff
@@ -0,0 +1,48 @@
+using Microsoft.AspNet.Mvc;
+using OmniSharp.AspNet5;
+using OmniSharp.Models;
+using OmniSharp.MSBuild;
+using System.Linq;
+
+namespace OmniSharp
+{
+    public class CurrentProjectController
+    {
+        private readonly AspNet5Context _aspnet5Context;
+        private readonly OmnisharpWorkspace _workspace;
+        private readonly MSBuildContext _msbuildContext;
+
+        public CurrentProjectController(AspNet5Context aspnet5Context, MSBuildContext msbuildContext, OmnisharpWorkspace workspace)
+        {
+            _aspnet5Context = aspnet5Context;
+            _msbuildContext = msbuildContext;
+            _workspace = workspace;
+        }
+
+        [HttpPost("currentprojectcommands")]
```

Comment:
I started to put this under /project to mirror the same response as
/projects with just the current project in the response however I'm not
sure what you mean by `Also don't parse the project name and assume the
target framework. We have a structure for that.` I continued with my
substring approach but then couldn't assign `AspNet5WorkspaceInformation`
projects property once I filtered it out.

I'm not that fussed about this at the moment so please clone this branch
and tidy it up.